corehq/apps/userreports/models.py

```diff
@@ -462,6 +464,16 @@ def all(cls):
                     doc['_id'] = cls.get_doc_id(domain, doc['table_id'])
                     yield DataSourceConfiguration.wrap(doc)
 
+        for provider_path in settings.STATIC_DATA_SOURCES:
+            try:
+                provider_fn = to_function(provider_path, failhard=True)
+            except:
```

Comment:
should probably at least only trap `Exception`. what's the use case for not always failing hard even outside of debug mode?